packages/create/init.ts

```diff
@@ -98,6 +107,27 @@ function mergeGitIgnore(existingGISet: Set<string>, templateGISet: Set<string>):
     return joined.replaceAll('\n#','\n\n#');
 }
 
+function updateAppConfig(configFilePath: string, appName: string): void {
+  try {
+    const content = fs.readFileSync(configFilePath, 'utf-8');
+    const configFile = YAML.parse(content) as ConfigFile;
+    // Change the app_db_name field to the sanitized appName, replacing dashes with underscores
+    let appDbName = appName.replace('-', '_');
+    if (appDbName.match(/^\d/)) {
+      appDbName = "_" + appDbName; // Append an underscore if the name starts with a digit
+    }
```

Comment:
If the app name was 30 characters and becomes 31, this will fail during cloud deploy

Reply:
This is app DB name not app name.